Lib/mzscheme/cdata.i

```diff
@@ -0,0 +1,10 @@
+/* -----------------------------------------------------------------------------
+ * cdata.i
+ *
+ * SWIG library file containing macros for manipulating raw C data.
+ *
+ * TODO:
+ *  Use generic cdata bu deleting this file and implementing
```

Comment:
Typo, presumably should be "by"?  Repeated below.

Reply:
Yap